automation/vars/uni05epsilon.yaml

```diff
@@ -0,0 +1,41 @@
+---
+vas:
+  uni05epsilon:
+    stages:
+      - path: examples/dt/uni05epsilon/control-plane/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+            --timeout=60s
+        values:
+          - name: network-values
+            src_file: values.yaml
+        build_output: nncp.yaml
+
+      - path: examples/dt/uni05epsilon/control-plane
+        wait_conditions:
+          - >-
+            oc -n openstack wait openstackcontrolplane
+            controlplane
+            --for condition=Ready
+            --timeout=30m
+        values:
+          - name: network-values
+            src_file: nncp/values.yaml
+          - name: service-values
+            src_file: service-values.yaml
+        build_output: control-plane.yaml
+
+      - path: examples/dt/uni05epsilon
+        wait_conditions:
+          - >-
+            oc -n openstack wait openstackdataplanedeployment
+            edpm-deployment
+            --for condition=Ready
+            --timeout=2400s
+        values:
+          - name: edpm-nodeset-values
+            src_file: values.yaml
+        build_output: edpm.yaml
```

Comment:
This should be broken up into two steps, one for `OpenStackDataPlaneNodeSet` and then another for `OpenStackDataPlaneDeployment`.  Doing this will require changes elsewhere in the directory structure for this DT.  Please see [1] for reference.  If we want to do this in a follow-up, that is fine, but someone should commit to it.  Unialpha01 DT was allowed to merged without this but it still needs the dataplane split, and we don't want to get into the habit of letting it slide.

[1] https://github.com/openstack-k8s-operators/architecture/blob/main/automation/vars/uni04delta.yaml

Reply:
Roger, roger, let me see what I can do about it...

Reply:
Ugh... So... I did it... I think (?)

But I have no idea what I did, so let me test it now again :upside_down_face:

Reply:
In `dt/uni05epsilon/deployment` I see a separate deployment type created as you documented.
```
[fultonj@runcible deployment{Uni-Epsilon}]$ kustomize build .
apiVersion: dataplane.openstack.org/v1beta1
kind: OpenStackDataPlaneDeployment
metadata:
  name: edpm-deployment
  namespace: openstack
spec:
  nodeSets:
  - openstack-edpm
[fultonj@runcible deployment{Uni-Epsilon}]$ 
```

Reply:
Ok, it apparently succeeded and reached the tempest phase, so it is better than I expected.
However, tempest fails due to missing credentials (?) and I am not sure if I missed something or the problem is somewhere else (there was no such problem before I made the split).
```
tempest.lib.exceptions.InvalidCredentials: Invalid Credentials
Details: Credentials are: {'domain_id': None, 'domain_name': 'Default', 'username': None, 'project_domain_id': None, 'project_domain_name': 'Default', 'project_id': None, 'project_name': None, 'tenant_id': None, 'tenant_name': None, 'user_domain_id': None, 'user_domain_name': 'Default', 'user_id': None, 'system': None} Password is not defined.
```

Reply:
The NodeSet deployment split should not be related to this.

Reply:
Ok, the deploment seem to went fine (no complains from pods observed, tempest tests reached).